unit_test/test_spdm_requester/key_update.c

```diff
@@ -3253,6 +3253,8 @@ libspdm_return_t libspdm_requester_key_update_test_receive_message(
                          secured_message_context->aead_iv_size);
         secured_message_context->application_secret
         .response_data_sequence_number = curr_rsp_sequence_number;
+        secured_message_context->application_secret_backup
+        .response_data_sequence_number = 0;
```

Comment:
I am not sure why we need reset to 0 in decode.

We need figure out why encode sequence is 0, in second message.